inst/qml/msaGaugeRRnonrep.qml

```diff
@@ -105,127 +105,128 @@ Form
 		}
 	}
 
-	Section
+	Group
 	{
-		title: 									qsTr("Gauge r&R Options")
+		title: 								qsTr("Analysis options")
 
-		Group
+		DropDown
 		{
-			title: 								qsTr("Analysis Options")
+			name: 							"processVariationReference"
+			label: 							qsTr("Std. dev. reference")
+			id: 							variationReference
+			indexDefaultValue: 				0
+			values: [
+				{ label: qsTr("Study std. dev."), value: "studySd"},
+				{ label: qsTr("Historical process std. dev."), value: "historicalSd"}
+			]
+		}
 
-			DropDown
-			{
-				name: 							"processVariationReference"
-				label: 							qsTr("Std. Deviation reference")
-				id: 							variationReference
-				indexDefaultValue: 				0
-				values: [
-					{ label: qsTr("Study Std. Deviation"), value: "studySd"},
-					{ label: qsTr("Historical process Std. Deviation"), value: "historicalSd"}
-				]
-			}
+		DoubleField
+		{
+			name:							"historicalSdValue"
+			label:							qsTr("Std. dev. value")
+			defaultValue:					3
+			enabled:						variationReference.currentValue == "historicalSd"
+		}
+
+		CheckBox
+		{
+			name:							"tolerance"
+			label:							qsTr("Tolerance width")
+			childrenOnSameRow:				true
 
 			DoubleField
 			{
-				name:							"historicalSdValue"
-				label:							qsTr("Std. Deviation value")
-				defaultValue:					3
-				enabled:						variationReference.currentValue == "historicalSd"
+				name: 						"toleranceValue"
+				defaultValue:				1
+				min:						0.000000001
```

Comment:
why not
```suggestion
				min:						0
				inclusive:					JASP.MaxOnly
```
?

See https://github.com/jasp-stats/jasp-desktop/blob/development/Docs/development/jasp-qml-guide.md#doublefield for the inclusive option and its possible values.

Reply:
Yes, thanks!